CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.14...3.22)
 
 project(rebuild LANGUAGES Fortran)
 
+
+add_subdirectory(mod)
+add_subdirectory(smod2)
```

Comment:
Please add the subdir `CMakeLists.txt` files to this PR.

Reply:
woops, I think I pushed them in now